testbed/testbed/receivers.go

```diff
@@ -187,71 +187,90 @@ func (jr *JaegerDataReceiver) ProtocolName() string {
 }
 
 // OTLPDataReceiver implements OTLP format receiver.
-type OTLPDataReceiver struct {
+type baseOTLPDataReceiver struct {
 	DataReceiverBase
+	protocol        string
```

Comment:
What are the expected values? Is this going to contain "grpc" or "http"? It may be more suitable to call this a `transport`.

UPDATE: looks like this needs to match the exporter name. Maybe call this `exporter` instead? Otherwise I was confused and did not expect it to contain the "otlp" and "otlphttp" values. At the minimum please document, best to use a strong type with possible valid values defined as consts.

Reply:
Renamed.